docs/UserGuide.md

```diff
@@ -183,6 +183,16 @@ Examples:
 * `list` followed by `memo 2 m/meet at 6pm` adds the memo “meet at 6pm” to the 2nd person in the address book. 
 * `find Betsy` followed by `memo 1 m/meet at 6pm` adds the memo “meet at 6pm” to the 1st person in the results of the `find` command.
 
+### Invoke recent command :`invoke`
+
+Invoke the most recent command being used.
+
+Format: `invoke`
+* Invokes the most recent command and autofill it in the textbox.
+
+Examples:
+* if the last command is `memo 2 m/meet at 6pm`, `invoke` autofills the textbox with `memo 2 m/meet at 6pm`.
```

Comment:
Shouldn't "if" be capitalized? In addition, I think "previous command" is more descriptive than "last", consider changing it if you think so too?